Dockerfile-centos-7

```diff
@@ -26,12 +26,7 @@ RUN source /opt/rh/rh-ruby23/enable && \
   /opt/rh/rh-ruby23/root/usr/bin/gem install -q --no-ri --no-rdoc -v $FPM_VERSION fpm
 RUN yum install -y python${PYTHON_VERSION}
 
-WORKDIR /pd-agent-install
-COPY . /pd-agent-install
-RUN  mkdir -p /pd-agent-install/build-linux/release
-RUN  cd /pd-agent-install/build-linux && ./make_rpm.sh /pd-agent-install/build-linux/gnupg /pd-agent-install/build-linux/release
-RUN  echo "Installing rpm package"
-
-RUN ./scripts/build.sh centos
+COPY . $DOCKER_WORKDIR
+WORKDIR $DOCKER_WORKDIR
```

Comment:
Part of this decision was to support interactive builds, something required for production releases with GPG passphrases.